configs/15.0/dockerfile-devel

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG ARCH
+ARG DISTRO_NAME
+ARG DISTRO_NICK
+
+FROM ${ARCH}/${DISTRO_NAME}:${DISTRO_NICK}
```

Comment:
This don't work with x86_64. At least for docker.io registry there is no x86_64/debian:buster.
It works fine on ppc64le.

Reply:
Ubuntu and Debian use `amd64` instead of `x86_64`.
Fixed.